students/mark_mcduffie/Lesson3/list_lab.py

```diff
@@ -0,0 +1,78 @@
+#Mark McDuffie
+#Lesson3
+#list lab
+
+def seq1(list):
+#let user add fruit to the list
+    fruit = input("What fruit would you like to add? ")
+    list.append(fruit)
+    print(list)
+#index fruit list and let user select a fruit
+    number = input("What number of fruit would you like to select? ")
+    fruitIndex = list[int(number) - 1]
```

Comment:
Most things (except classes) should be snake_case in python.  So fruit_index, as well as function names.